.github/workflows/github_actions.yml

```diff
@@ -57,6 +57,7 @@ jobs:
       matrix:
         arch: [armv7, aarch64]
         cxx_compiler: [g++-10, clang++-11]
+        feature: ["none", "crypto"]
```

Comment:
It seems `uraimo/run-on-arch-action@v2.5.0` doesn't allow us to set an empty element in the matrix, so I use a `none` element to represent the situation